contracts/dao/FeiDAOTimelock.sol

```diff
@@ -39,4 +42,16 @@ contract FeiDAOTimelock is Timelock, CoreRef {
         pendingAdmin = newAdmin;
         emit NewPendingAdmin(newAdmin);
     }
+
+    /// @notice one-time option to roll back the Timelock to old timelock
+    /// @dev guardian-only, and expires after the deadline. This function is here as a fallback in case something goes wrong.
+    function rollback() external onlyGuardianOrGovernor {
+        require(block.timestamp <= ROLLBACK_DEADLINE, "FeiDAOTimelock: rollback expired");
+        
+        IFeiDAO(admin).updateTimelock(OLD_TIMELOCK);
```

Comment:
Does this pass the onlyGovernance check on updateTimelock?

Reply:
Other than this looks good

Reply:
it does, onlyGovernance requires the msg.senser to be itself if no timelock, otherwise the timelock

Reply:
Will confirm in e2e